frameworks/person-escort-record/questions/prescribed-medication.yml

```diff
@@ -1,14 +1,24 @@
 type: radio
-question: Does this person need or have they been prescribed any medication?
+question: Do they need or have they been prescribed any other medication not required while moving?
 hint: This includes relevant regular medication that they need or already have with them, as well as anything that they may have been prescribed whilst in custody
 description: Prescribed medication
 options:
   -
     label: "Yes"
     value: "Yes"
+    followup:
+      - prescribed-medication-carrier
     followup_comment:
       label: Provide details of medication
-      hint: Enter any medication that is required while moving and include medication name, direction, dosage, frequency, time last given and time next given for each medication
+      hint: |
+        Ensure the following details are included:
```

Comment:
Same as other comment. I think `Ensure` can be removed to make this instruction clearer.